bin/wwwtxt

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+require "gli"
+require "wwwtxt"
+
+include GLI::App
+
+program_desc "Turn websites into plain text"
+
+version WwwTxt::VERSION
+
+desc "Download a single URL as plain text"
+flag [:o,:out_dir], default_value: File.join(ENV["HOME"], "texts")
```

Comment:
don't really like this as a global option, but I can't figure out gli's subcommand flags...

eventually I'm going to prefer setting the output directory from a config file (`~/.wwwtxt`)